phpcs.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0"?>
 <ruleset>
 	<file>src/</file>
-	<file>tests/</file>
+	<file>tests/phpunit</file>
+
+	<arg name="extensions" value="php"/>
```

Comment:
What is this for?

Reply:
phpcs was linting the JS files: https://github.com/ProfessionalWiki/WikibaseFacetedSearch/actions/runs/12794862693/job/35670904144

This is added to make sure that phpcs only touches PHP files

Reply:
Is this still necessary if the paths are limited to (PHP) src and tests?